### ISSUE TITLE
Will this module be updated for 0.8.x ?

### ISSUE THREAD

**Issue by jstebenne** [DasSauerkraut/calendar-weather#213]
I really love this module and I don't want to lose it when upgrading.

Is it going to be updated or will another maintainer be assigned? I might be willing to make a pull request here if I know it will be reviewed and merged. I'd like to avoid making a separate module just so people can continue using it.

**Comment by jstebenne** [comment]
I just saw in the readme that the project is no longer maintained.

@DasSauerkraut Would it be possible to transfer ownership of the repo? I would like to pick it up to continue its development and review pull requests.

**Comment by Geekswordsman** [comment]
I have gotten it working now in v0.8.x - it thankfully is an easy fix to keep the current functionalities.  If ownership could be taken over by someone else, that would be great - otherwise, we'll have to create a 'new' module and publish it that way, meaning all existing settings will not carry over for anyone that updates to it.

At the very least they can export old prior to updating, and importing into this one.

**Comment by jstebenne** [comment]
Can you send the changes as a pull request? I'll review it to get familiar with it and we can increase activity in the repo this way. I really want to avoid having to create a new module and make people search for it.

I will probably start looking at the current open issues and see what I can start working on.

**Comment by Geekswordsman** [comment]
I already committed to where there are lots of changes making it ready for a new module; let me create a new branch with *just* the small changes required (the module is still backwards compatible with v0.7.x!) and get that pull request in.  You'll see it in just a bit :)

**Comment by Geekswordsman** [comment]
There we go.  When I said it was a simple fix, I mean it.  The only reason that I can find this module broke is because of a change in FormApplication in core Foundry - in which there's clearly a bug (I reported it on the discord).  If it wasn't for that bug, this module likely wouldn't be broken in the first place.

I've tested calender-weather with this fix now utilizing Foundry v0.8.5 and 5e v1.3.0.  I was able to load the default calendar, adjust dates, adjust the clock using the widget, generate weather, see lighting changes in a scene that it was enabled to modify, and see weather effects activate.

**Comment by akrigline** [comment]
Since this module is licensed under Apache, I believe its fair game for you to fork it and continue its work. Submitting a ticket to Foundry Support will let you kick off the process to transfer the listing. Obviously this all works best with the Original Author's sign-off. Reaching out on discord might be the easiest way to get their blessing.

**Comment by jstebenne** [comment]
Thanks for the info, I didn't know we could request a transfer for the listing.

I've already made a fork in a new repository here: https://gitlab.com/jstebenne/foundryvtt-calendar-weather

Ideally I would like to have multiple admin/maintainers on the repo to mitigate the bus factor. I will add references to this repo here on github so we can start working on the pending issues.

@Geekswordsman Can you also send a pull request on the GitLab repo so I can merge it and add you as a maintainer (if you would like to be one)

**Comment by Geekswordsman** [comment]
Yeah, I'll get that done as soon as I can.  And I don't mind being a maintainer :)

**Comment by ctbritt** [comment]
Just curious as to when a new version might be available. I've changed the files, as per @Geekswordsman's suggestions, but it didn't really work right. Settings box came out looking like this: 
<img width="605" alt="Screen Shot 2021-06-03 at 5 37 57 PM" src="https://user-images.githubusercontent.com/4516034/120715355-8163c200-c492-11eb-8bf0-ece4a8d2b87e.png">

**Comment by Geekswordsman** [comment]
@ctbritt I think you have something missing or not loading correctly; just replacing the html templates and loading the Calendar without issue.

Foundry: v0.8.6
AboutTime v0.8.4
Calendar-Weather v3.1.2
CoC v0.6.0 
DnD5e 1.3.3
![Calendar-Weather-Template-Fix](https://user-images.githubusercontent.com/41522046/120722868-8af32700-c49e-11eb-8901-bd2f3eaadc65.PNG)

**Comment by jstebenne** [comment]
For those stumbling upon this issue in the future, ownership of the module has been transferred to me (The Bird#8334 on Discord, Julien Stébenne (@jstebenne) on GitLab).

You can follow and contribute on the GitLab repository: https://gitlab.com/jstebenne/foundryvtt-calendar-weather
